proto/errorpb.proto

```diff
@@ -128,6 +128,11 @@ message DataIsNotReady {
     uint64 safe_ts = 3;
 }
 
+message RecoveryInProgress {
+    // The requested region ID
+    uint64 region_id = 1;
```

Comment:
Isn't it part of the request?

Reply:
nope, the request has the context, but the response doesn't.

Reply:
What I mean client should know the request corresponding to the response, so it should know the region id.

Reply:
Yes, but seems the error `NotLeader`, `RegionNotFound` and etc. includes the region id as well